content/source/layouts/enterprise2.erb

```diff
@@ -202,6 +202,9 @@
           <li<%= sidebar_current("docs-enterprise2-private-installer") %>>
             <a href="/docs/enterprise/private/install-installer.html">Installation (Installer Beta)</a>
             <ul class="nav">
+              <li<%= sidebar_current("docs-enterprise2-private-rhel") %>>
+                <a href="/docs/enterprise/private/rhel-istall-guide.html">RedHat Enterprise Install Guide</a>
```

Comment:
🚨  `rhel-istall-guide` 🚨 broken link

Fixed in https://github.com/hashicorp/terraform-website/pull/182 /cc @evanphx